src/xenia/kernel/xboxkrnl/xboxkrnl_xconfig.h

```diff
@@ -0,0 +1,40 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2022 Ben Vanik. All rights reserved.                             *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+
+#ifndef XENIA_KERNEL_XBOXKRNL_XBOXKRNL_XCONFIG_H_
+#define XENIA_KERNEL_XBOXKRNL_XBOXKRNL_XCONFIG_H_
+
+#include "xenia/kernel/util/shim_utils.h"
+#include "xenia/xbox.h"
+
+enum xcConsoleRegion {
+  xcConsoleRegion_US,
+  xcConsoleRegion_Japan,
+  xcConsoleRegion_Asia,
+  xcConsoleRegion_Europe,
+  xcConsoleRegion_RegionFree,
+};
+
+enum xcRegionSettingType {
```

Comment:
Same as above, enum class is preferred (and PascalCase beginning with k for the enumerations "kUserLanguage")